src/content/en/2020/pwa.md

```diff
@@ -4,21 +4,437 @@ part_number: II
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRRpTSA4fsHwUap-ByQ08j95uo7Zm1kY6lTSvA-DZT54g2QZ0guV7db3QyQwQgMPzsKsJ43gbzqfJst/pubchart?oid=257125618&format=interactive",
+  sheets_gid="887021927",
+  sql_file="top_manifest_properties_sw.sql"
+  )
+}}
+
+### Top Manifest display values
+
+
+{{ figure_markup(
+  image="pwa-most-used-display-values-for-service-worker-pages.png",
+  alt="Most used display values for service worker pages.",
+  caption="Most used `display` values for service worker pages.",
+  description="Bar chart showing the percenatge of pages using the 5 most common `display` values, dominated by `standalone` which is used by 86.73% of desktop and 89.28% of mobile pages, `minimal-ui` is used by 6.30% of desktop and 5.00% of mobile, `fullscreen` by 4.80% of desktop and 4.11% of mobile, 1.15% of desktop pages and 0.88% of mobile pages do not set a `display` value, and finally 1.00% of desktop and 0.72% of mobile set this to `browser`.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRRpTSA4fsHwUap-ByQ08j95uo7Zm1kY6lTSvA-DZT54g2QZ0guV7db3QyQwQgMPzsKsJ43gbzqfJst/pubchart?oid=872942071&format=interactive",
+  sheets_gid="1208193001",
+  sql_file="top_manifest_display_values_sw.sql"
+  )
+}}
+
+### Top manifest categories
+
+Note these numbers are different than 2019 version as we are looking into PWA pages (i.e. Manifests and SW) as it makes more sense to concentrate on the PWA stats and ignoring the non-PWA for this section.
+
+{{ figure_markup(
+  image="pwa-manifest-categories.png",
+  caption="PWA manifest categories.",
+  description="Bar chart showing the percentage of PWA pages for the top 10 most popular categories with shopping have 3.45% of desktop but a huge 13.46% of mobile pages towering above the other figures. After that news has 4.60% of desktop and 5.26% of mobile, entertainment has 6.90% and 5.26% respectively, utilities have 6.32% and 4.74%, business has 6.90% and 4.74%, lifestyle having 2.87% and 3.16, social 2.87% and 2.63%, finance having 2.87% and 2.63% and finally web having 1.72% and 2.11%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRRpTSA4fsHwUap-ByQ08j95uo7Zm1kY6lTSvA-DZT54g2QZ0guV7db3QyQwQgMPzsKsJ43gbzqfJst/pubchart?oid=137093819&format=interactive",
+  sheets_gid="423855457",
+  sql_file="top_manifest_categories_sw.sql"
+  )
+}}
```

Comment:
Again the chart is presented without any analysis or interpretation for the reader.

Reply:
I remember adding more info for this! Will double check the docs and add if this is missed.